<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_info</class>
 <widget class="QWidget" name="choose_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>606</x>
        <y>89</y>
        <width>567</width>
        <height>838</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>空间信息设置</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>微软雅黑 Light</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>请设置空间总面积：（70-200）</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tot_area">
               <property name="font">
                <font>
                 <family>微软雅黑 Light</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>微软雅黑 Light</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>请选择房间数量：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="room_num">
               <property name="font">
                <font>
                 <family>微软雅黑 Light</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请输入房间总面积：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="room_area">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请输入客厅总面积：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="livingroom_area">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请选择墙体材料：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="material_id">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>粘土多孔砖（0.58）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>灰砂砖240（1.10）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>炉渣砖240（0.81）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>煤矸石烧结砖（0.63）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>粉煤灰烧结砖（0.50）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>混凝土双排孔砌块190（0.68）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>混凝土多孔砖（0.80）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>混凝土砌块内填膨胀珍珠岩（单排孔）（0.33）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>煤矸石砌块内填膨胀珍珠岩（0.27）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ALC加气混凝土砌块（0.20）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>粉煤灰加气混凝土砌块（0.22）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>页岩模数烧结砖（0.45）</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请输入窗户数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="window_num">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请选择房间朝向：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="oriention">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>东西走向</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>南北走向</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>请输入墙体厚度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <family>微软雅黑 Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>半砖墙（115mm）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>一砖墙（240mm）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>一砖半墙（365mm）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>二砖墙（490mm）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3/4砖墙（180mm）</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>label</zorder>
      <zorder>verticalSpacer_8</zorder>
      <zorder>verticalSpacer_9</zorder>
      <zorder>verticalSpacer_10</zorder>
      <zorder>verticalSpacer_11</zorder>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>1600</x>
        <y>770</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/逻辑图标/逻辑小图标/下一页按钮.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1675</x>
        <y>30</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/逻辑图标/逻辑小图标/音乐.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_button">
      <property name="geometry">
       <rect>
        <x>1675</x>
        <y>30</y>
        <width>75</width>
        <height>75</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/逻辑图标/逻辑小图标/暂停音乐.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
